src/ds/ds-color-common.cpp

```diff
@@ -66,8 +68,7 @@ namespace librealsense
                 gain_option,
                 auto_exposure_option));
 
-        // Starting with firmware 5.10.9, auto-exposure ROI is available for color sensor
-        if (_fw_version >= firmware_version("5.10.9.0"))
+        if( _color_ep.supports_feature( auto_exposure_roi_feature().get_name() ) )
```

Comment:
I don't think we should assume that the interfaces are not abstract.

Reply:
Done. Using static IDs.